lib/fizzy/limits.hpp

```diff
@@ -0,0 +1,9 @@
+namespace fizzy
+{
+namespace
```

Comment:
Anonymous namespace should never be in a header.

Reply:
That was a typo, not sure why it even compiled.

Reply:
The effect on `constexpr` variables are neglectable - they have internal linkage anyway.